cssdata.py

```diff
@@ -0,0 +1,112 @@
+# import from existing libraries
+import numpy as np
+import pandas as pd
+from matplotlib import pyplot as plt
+
+# import from my modules
+from dataimportingfunctions import get_data_dir_list
+
+def exp_num_list():
+    """"
+    define experiment numbers that you want to include from the whole data
+    """
+
+    exp_num_list = [7, 8, 9, 10]
+    return exp_num_list
+
+
+def relative_density():

```

Comment:
Again, these are specific to your dataset and must be stored in a file and read, not as a function